apps/vor/pkg/postgres/postgres.go

```diff
@@ -235,3 +237,17 @@ type Weekday struct {
 	Day     time.Weekday `pg:",pk,use_zero"`
 	Class   Class
 }
+
+type LessonPlan struct {
+	Id          string `pg:"type:uuid"`
+	Title       string
+	Description string
+	ClassId     string `pg:"type:uuid"`
+	Repetition  int
+}
+
+type Attachment struct {
```

Comment:
Naming this attachment is a bit ambiguous I think (feels like it raises questions such as "what attachment? attached to what?"). Maybe FIle is a better name?.

Also, lesson plans would have many to many relationships with attachments.

Reply:
yes, it's still WIP. will add this later